src/Database/Franz/Network.hs

```diff
@@ -171,72 +181,95 @@ getResponse = do
                 return vres
 
 -- | Fetch requested data from the server.
--- Termination of the continuation cancels the request, allowing flexible
--- control of its lifetime.
+--
+-- Termination of 'fetch' continuation cancels the request, allowing
```

Comment:
I guess this shouldn't say "continuation" any more... Perhaps I should improve the docs a little bit.